msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerAccountManagerStrategy.java

```diff
@@ -135,7 +134,7 @@ void hello(@NonNull final OperationParameters parameters)
             throws BaseException {
 
         if (!BrokerMsalController.isAccountManagerPermissionsGranted(parameters.getAppContext())) {
-            throw new BrokerCommunicationException("Account manager permissions are not granted");
+            throw new BrokerCommunicationException("Account manager permissions are not granted", null);
```

Comment:
Minor nit regarding comments: flips between `AccountManager` and `Account manager`